src/@types/Shortcut.ts

```diff
@@ -71,6 +73,40 @@ interface Shortcut {
 
   // an icon that represents the shortcut in the Toolbar
   svg: (icon: Icon) => React.ReactNode
+
+  /** Multicursor support. If 'ignore', the shortcut will be executed as if there were no multicursors. When true, the shortcut will be executed for each cursor. Optional object for more control. */
+  multicursor:
+    | 'ignore'
+    | boolean
+    | {
+        /** Whether multicursor mode is enabled for this shortcut. */
+        enabled: boolean
+        /** An error message to display when multicursor mode is not enabled. */
+        error?: (getState: () => State) => string | null
+        /** Optional override for executing the shortcut for multiple cursors. */
+        execMulticursor?: (
+          cursors: Path[],
+          dispatch: Dispatch,
+          getState: () => State,
+          e: Event | GestureResponderEvent | KeyboardEvent | React.MouseEvent | React.TouchEvent,
+          { type }: { type: ShortcutType },
+          execMulticursor: () => void,
```

Comment:
Let's give this a different name than the property it's in. I would just call it `exec`, but we could also call it `execAll` if you wanted to differentiate it from the base shortcut property.